modules/va_notify/app/controllers/va_notify/callbacks_controller.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+module VANotify
+  class CallbacksController < VANotify::ApplicationController
+    include ActionController::HttpAuthentication::Token::ControllerMethods
+
+    service_tag 'va-notify'
+
+    skip_before_action :verify_authenticity_token, only: [:create]
+    skip_before_action :authenticate, only: [:create]
+
+    before_action :authenticate_callback
+
+    def create
+      Rails.logger.info { "Notification received: #{params.inspect}" }
+      notification_id = params[:id]
+
+      if (notification = VANotify::Notification.find_by(notification_id: notification_id))
+        Rails.logger.info("va_notify callbacks - Updating notification #{notification.id}")
+        notification.update(notification_params)
+      else
+        Rails.logger.info("va_notify callbacks - Received update for unknown notification #{notification_id}")
+      end
+
+      render json: { message: 'success' }, status: :ok
+    end
+
+    private
+
+    def authenticate_callback
+      authenticate_token || authenticity_error
+    end
+
+    def authenticate_token
+      authenticate_with_http_token do |token|
+        return false if bearer_token_secret.nil?
+
+        ActiveSupport::SecurityUtils.secure_compare(token, bearer_token_secret)
+      end
+    end
+
+    def authenticity_error
+      Rails.logger.info('va_notify callbacks - Failed to authenticate request')
+      render json: { message: 'Unauthorized' }, status: :unauthorized
+    end
+
+    def bearer_token_secret
+      Settings.dig(:va_notify, :status_callback, :bearer_token)
```

Comment:
I don't see a va_notify section in config/settings.yml.

Reply:
@rmtolmach I have added va_notify section. Thanks for your comment

Reply:
One more thing! Can you change it to dot notation since you know it'll always exist now? Steven _just_ put up a PR to change many other instances https://github.com/department-of-veterans-affairs/vets-api/pull/19099 so I want to make sure this is keeping with the new standard 😄